modules/thermal_hydraulics/test/tests/problems/brayton_cycle/closed_brayton_cycle.i

```diff
@@ -22,9 +22,9 @@ motor_torque_max = 400.0
 I_generator = 1.0
 generator_torque_per_shaft_speed = -0.00025
 
-motor_ramp_up_duration = 100.0
-motor_ramp_down_duration = 100.0
-post_motor_time = 100.0
+motor_ramp_up_duration = 1.0
+motor_ramp_down_duration = 1.0
+post_motor_time = 1.0
```

Comment:
Please just modify these in the test specs, not in the input file. I have a tutorial associated with this example that uses the 100 s ramps.